capstone/src/main/java/com/google/sps/servlets/announcements/AnnouncementsServlet.java

```diff
@@ -78,4 +82,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     response.sendRedirect("/about-us.html?name=" + clubName + "&tab=announcements");
   }
+
+  private Club getClub(DatastoreService datastore, String clubName) {
+    Query query =
+        new Query("Club")
+            .setFilter(
+                new FilterPredicate(Constants.PROPERTY_NAME, FilterOperator.EQUAL, clubName));
+    Entity entity = datastore.prepare(query).asSingleEntity();
+    if (entity == null) {
+      return null;
+    }
+    return new Club(
+        entity.getProperty(Constants.CLUB_NAME_PROP).toString(),
+        ImmutableList.copyOf((ArrayList<String>) entity.getProperty(Constants.MEMBER_PROP)),
+        ImmutableList.copyOf((ArrayList<String>) entity.getProperty(Constants.OFFICER_PROP)),
+        entity.getProperty(Constants.DESCRIP_PROP).toString(),
+        entity.getProperty(Constants.WEBSITE_PROP).toString());
```

Comment:
FYI, I changed the Club object so that it includes a String logo which represents the club logo blobkey. Just a warning that we might have to update this soon.

Reply:
Got it, thanks!